test/minikube/minikube_domain_resync_test.go

```diff
@@ -101,6 +101,55 @@ func checkArchives(t *testing.T, namespaceName string, adminPod string, numExpec
 	return
 }
 
+func TestAdminScaleDown(t *testing.T) {
```

Comment:
should this not be skipped if this is not the ENTERPRISE test suite?

Reply:
Yes. The file has tag "enterprise".